rwatch/ui/layer/menu_layer.c

```diff
@@ -384,6 +386,14 @@ static void menu_layer_update_proc(Layer *layer, GContext *nGContext)
 {
     MenuLayer *menu_layer = (MenuLayer *) layer->container;
     GRect frame = layer_get_frame(layer);
+    
+#ifndef PBL_RECT
+    if (!menu_layer->callbacks.draw_background)
```

Comment:
was the background draw callback not invoked previously? Is this is a bonus bugfix?

Reply:
It was invoked in the cell_draw. I moved the drawing of the background to the `update_proc` only if they didn't set that callback, and if on Chalk. Otherwise it looks strange in the animation because it didn't stay in place.

Reply:
Yes this behaviour of the cursor is unique to the chalk menus. Nonetheless I think that this is not the way to go. For one this is more consistent with the code, as in `draw the cell (background)` in the function called `_draw_cell` except for when running on a certain platform. Secondly this functionality ensured that there is a background for every cell instead of just drawing the background for the highlighted one.
Maybe we should draw the cell backgrounds (if no callback is set) for all cells via two or three calls to `graphics_fill_rect` for performance reason, but we should certainly not insert more special cases than needed.